hips/draw/simple.py

```diff
@@ -63,6 +63,10 @@ def __init__(self, geometry: WCSGeometry, hips_survey: HipsSurveyProperties, til
         self._tiles = None
 
     @property
+    def image(self):
```

Comment:
Always add a one-line docstring mentioning what it is, what object type is returned, and also add a type annotation.